sdks/how-to/how-to-migrate-to-icure-premium-features.md

```diff
@@ -0,0 +1,65 @@
+---
+slug: how-to-migrate-to-icure-premium-features
+description: How to Migrate to iCure Premium Features
+tags:
+- Open Source
+- Migration
+- Premium Features
+---
+
+# How to Migrate to iCure Premium Features
+
+:::note
+
+This how-to assumes that you have been using the iCure Medical Device SDK in your application with the open-source version
+ of the iCure backend and that, after creating an application, a database, and a suer using the iCure Cockpit, you now
+ want to use the cloud version of iCure.
+
+:::
+
+If you were using the open source version of the iCure backend, and you now are migrating to the cloud version, there are
+some things that you have to take into account. You will have access to additional features available only in the cloud version,
+but you will also have to make some changes in your code.
+
+## Changes
+
+The following steps are mandatory changes that you must perform to ensure that your application will work as intended.
+
+### iCure URL
+
+First, you will have to change the URL used to instantiate the MedTech API to allow you application to contact the iCure
+ Cloud Backend:
+```typescript
+const medtechApi = await medTechApi()
+    .withICureBaseUrl("https://kraken.svc.icure.cloud")
+    .withUserName(username)
+    .withPassword(password)
+    .withCrypto(webcrypto as any)
+    .build()
+```
+
+### Access Control
+
+Another important thing that you have to keep in mind is that the cloud version of the iCure backend applies a permission-based
```

Comment:
I would inverse the order of the explanation, like : 
```
While testing your app, you could experience some `403 - FORBIDDEN` errors on iCure Backend requests. This is because the cloud version applies a permission-based access control on its endpoints, which is useless for the iCure free version. 
To resolve those issues, make sure your data owner may access the requested data. If not, give him access thanks to the [giveAccessTo service](GIVE ACCESS TO DOC) first, or [share of all future data](AUTO DELEGATIONS DOC) with another user. 
```